### PR TITLE
feat: (CCOM-410) created the flag that is able to hide the last pagination

### PR DESCRIPTION
**Pull request by douglasjr97**
## Description
https://jirasoftware.catho.com.br/browse/QTM-410

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### A11y review
- [ ] A11y checks
  - [ ] accessible via keyboard?
  - [ ] Identifiable through assistive technology (screen-reader chrome extension)?
  - [ ] Semantically correct html elements (or if necessary identified by WAI-ARIA)?
  - [ ] Is there a deficiency in color contrast?

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile

### CODE REVIEW

File: components/Pagination/utils/pagination.js [hunk 1]
```diff
@@ -22,7 +22,11 @@ const getNumberPages = (active, total) => {
   return 3;
 };
 
-const pagination = ({ totalPages = 1, activePage = 1 } = {}) => {
+const pagination = ({
+  totalPages = 1,
+  activePage = 1,
+  showLastPagination = true,
```
Comment (rapahaeru) [inline components/Pagination/utils/pagination.js:28]: was this value already declared in Pagination.jsx?
i think its redundant here.

File: components/Pagination/sub-components/Desktop.jsx [hunk 2]
```diff
@@ -89,6 +91,7 @@ Desktop.defaultProps = {
   prevButtonText: 'Previous',
   handlePageClick: undefined,
   followOnlyFirstPage: false,
+  showLastPagination: true,
```
Comment (rapahaeru) [inline components/Pagination/sub-components/Desktop.jsx:94]: was this value already declared in Pagination.jsx?
i think its redundant here.

**Review by rapahaeru — CHANGES_REQUESTED** [review]

File: components/Pagination/sub-components/Desktop.jsx (hunk 2 above)
Comment (douglasjr97) [reply components/Pagination/sub-components/Desktop.jsx]: I do

File: components/Pagination/utils/pagination.js (hunk 1 above)
Comment (MarcosViniciusPC) [inline components/Pagination/utils/pagination.js:28]:
```suggestion
  showLastPagination,
```

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Coverage
[x] Unit tests
[x] Regression
[x] Code review

Browsers review
 [x] Layout review
 [x] Chrome
 [x] Safari
 [x] Mobile
 
 As a suggestion I have you add to the propTypes of desktop.jsx the value 'showLastPagination: PropTypes.bool.isRequired' and make the adjustment below

**Review by rapahaeru — APPROVED** [review]

**Pull request closed without merging**